### ISSUE TITLE
Not being able to teleport onto an InteractableObject

### ISSUE THREAD

**Issue by kylamon1** [thestonefox/SteamVR_Unity_Toolkit#449]
My original message to StoneFox:
"Hey I have an issue that I am not sure if it is a bug or I am misinterpreting what the buttons mean.
I have the floor of my level made up of intractable tiles. I have it so on my (right hand) simplePointer I can click and it does the desired action to build something. But on my lefthand I have the Bezier pointer for my teleport.
The pic below is the properties of my tile where my class clickableTile just extends you InteractableObject. http://imgur.com/a/jv3o1
I have the tile set to "Allow controller to use" as "right only" but when I try to teleport to a tile it will 'Use' the tile and construct something there(where the script then deletes the tile). Once the tile is gone I can teleport there.
tl;dr I cant teleport onto an interactable object. I think this might be a bug.
If not a bug how can I teleport onto an interactable object? Thanks."

As per TheStoneFox:
"It's not a bug, it's expected. If you have the "Pointer Activates Use Action" ticked on the object then the pointer will activate the use action it will prevent you from teleporting to it otherwise when you activated use with a pointer you'd teleport to that location all the time.
However, maybe it's better to always teleport you to that area and just disable the teleport action on the pointer if you don't want teleporting enabled.
It's worth raising issues like this on the github issues page as well: https://github.com/thestonefox/SteamVR_Unity_Toolkit/issues
Then you'll have more people looking over the problem :)"

Another possible solution:
Have another dropdown menu for the pointers style something like
Use Only, Teleport only, Use and Teleport

Another possible solution:
I dont know of anyone that uses the bezier pointer for "using" interactable objects.  It is almost exclusively used for teleport. Could we make a seperate subclass of worldPointer that still uses Bezier, but only teleports but will not use interactable objects.

**Comment by kylamon1** [comment]
Also possible bug in the picture above my interactable object is set to right only, but both controllers are able to 'use' the tile.

**Comment by thestonefox** [comment]
> I have the tile set to "Allow controller to use" as "right only" but when I try to teleport to a tile it will 'Use' the tile and construct something there(where the script then deletes the tile). Once the tile is gone I can teleport there.

I think this is where the bug lies. I think you're right if you don't have "use" on the controller then it shouldn't attempt to use it with the pointer. That's where I'll fix it :)

**Comment by thestonefox** [comment]
Resolved with https://github.com/thestonefox/SteamVR_Unity_Toolkit/pull/454

**Comment by kylamon1** [comment]
I checked and this does allow me to teleport onto my interactableObject tiles. So all is not as it is expected.

Thanks

**Comment by thestonefox** [comment]
Did you try the version in the pull request?

**Comment by kylamon1** [comment]
Stupid fingers...it should say all is NOW as it is expected. Its all good in the hood.

**Comment by thestonefox** [comment]
Cool, I'll merge that pr into the master branch then :)
